### DISCUSSION TITLE
Is pacman version of fzf outdated or abandoned?

### DISCUSSION THREAD

**Discussion by tony-sol** [junegunn/fzf#3965] [Q&A]
I've tried to install fzf via pacman without aur and use my personal config (from fzf installed via homebrew) and got
```
unknown option: --highlight-line
```

So, i've compared version and got:
via homebrew:
```
0.54.3 (brew)
```
via pacman:
```
0.42.0 (d471067e)
```

Are there any plans to make versions the same across package managers?

**Comment by junegunn** [comment]
I have no idea. You should contact the package maintainers. 

But according to repology, the latest arch linux provides the latest version of fzf.

[![Packaging status](https://repology.org/badge/vertical-allrepos/fzf.svg?columns=3)](https://repology.org/project/fzf/versions)
